src/test/java/com/kodilla/ecommercee/domain/GroupTestSuite.java

```diff
@@ -0,0 +1,86 @@
+package com.kodilla.ecommercee.domain;
+
+import com.kodilla.ecommercee.repository.GroupRepository;
+import com.kodilla.ecommercee.repository.ProductRepository;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+@SpringBootTest
+@RunWith(SpringRunner.class)
+public class GroupTestSuite {
+
+    @Autowired
+    GroupRepository groupRepository;
+    @Autowired
+    ProductRepository productRepository;
+
+    @Test
+    public void shouldAddToDataBase(){
+        //Given
+        Group group = new Group("Shoes");
+        long prevNumOfRecords = groupRepository.count();
+        //When
+        groupRepository.save(group);
+        //Then
+        long nextNumOfRecords = groupRepository.count();
+        assertEquals(1,nextNumOfRecords - prevNumOfRecords);
+        //Clean Up
+        groupRepository.deleteById(group.getId());
+    }
+
+    @Test
+    public void shouldFindAllGroup(){
+        //Given
+        long prevNumOfRecords = groupRepository.count();
+        Group group = new Group("Shoes");
+        Group group2 = new Group("Accessories");
+        groupRepository.save(group);
+        groupRepository.save(group2);
+        //When
+        long nextNumOfRecords = groupRepository.count();
```

Comment:
jeżeli chcemy wyszukać wszystkie grupy to powinnismy skrozystać z metody findAll()